### PR TITLE
Flower service and updated kubectl outputs in examples/celery-rabbitmq

### PR DESCRIPTION
**Pull request by mwielgus**

### CODE REVIEW

File: examples/celery-rabbitmq/README.md [hunk 1]
```diff
@@ -213,17 +239,22 @@ Again, it uses the Kubernetes-provided environment variable to obtain the addres
...
+`kubectl get service flower-service` will help you to get the external IP addresses of the flower service.
+
+```
+NAME             LABELS        SELECTOR                         IP(S)            PORT(S)
+flower-service   name=flower   app=taskQueue,component=flower   10.0.44.166      5555/TCP
+                                                                162.222.181.180
 ```
 
-Now you know on which host Flower is running (in this case, 10.245.1.4), you can open your browser and enter the address (e.g. `http://10.245.1.4:5555`. If you click on the tab called "Tasks", you should see an ever-growing list of tasks called "celery_conf.add" which the run\_tasks.py script is dispatching.
+Point your internet borowser to the appropriate flower-service address, port 5555 (in our case http://162.222.181.180:5555).
```
Comment (nikhiljindal) [inline examples/celery-rabbitmq/README.md]: borowser -> browser

**Comment by nikhiljindal** [comment]
cc @thockin @bgrant0607 @caesarxuchao as potential reviewers

File: examples/celery-rabbitmq/README.md (hunk 1 above)
Comment (mwielgus) [inline examples/celery-rabbitmq/README.md]: fixed

**Comment by caesarxuchao** [comment]
Thank you @mwielgus. The modification looks good. But you also need to add an entry in examples_test.go to validate the new flower-service.yaml to pass the tests.

**Comment by mwielgus** [comment]
@caesarxuchao Added.

**Comment by caesarxuchao** [comment]
Thanks @mwielgus. The rule now requires LGTM from two TL. Let's wait @bgrant0607 @thockin to take a look.

**Comment by caesarxuchao** [comment]
@mwielgus Shippable still fails. It seems you didn't push the changes.

**Comment by mwielgus** [comment]
@caesarxuchao  Shippable is green now.

**Comment by caesarxuchao** [comment]
LGTM. Need two TL to give LTGM.  @bgrant0607 @thockin.

**Comment by bgrant0607** [comment]
After 1.0.

**Comment by mwielgus** [comment]
@bgrant0607 Without this change people on GCE (or any other cloud provider) will not be able to see anything "working" in this example as the Flower web ui will be visible only inside of the cluster. Are you OK with this?

**Comment by bgrant0607** [comment]
LGTM
ok to merge

**Comment by caesarxuchao** [comment]
ref #10168

**Comment by piosz** [comment]
adding ok to merge label according to what @bgrant0607 wrote

**Pull request merged**